### ISSUE TITLE
Add Linux, Windows, Firefox support for browser-info

### ISSUE THREAD

**Issue by corevo** [SeleniumHQ/selenium-ide#822]
- [ ] Linux
- [ ] Windows
- [ ] Firefox

Linux is pretty similar and even easier than macOS, there is a `google-chrome` binary in `PATH` and it can be parsed to any of the channels. (`google-chrome --version`)

**Comment by corevo** [comment]
This [chromedriver code](https://cs.chromium.org/chromium/src/chrome/test/chromedriver/chrome/chrome_finder.cc?sq=package:chromium&dr=C&g=0) can help with finding the location of Chrome binaries in the system.

**Comment by dkindt** [comment]
I would like to pick this up. I have been reviewing code and it looks like you've added Linux support. Can we update this issue? It seems as tho there's just Windows and Firefox support left?

**Comment by corevo** [comment]
Yes I added Linux support because I just tested it on Docker, my original plan was to add github actions and test it on all systems (just these tests that depend on OS fs structure).

So the Windows task is a bit rough, I was lazy for no reason just because I use mac, but I should've setup github actions correctly, feel free to add Firefox, it should be similar, and if you want we can create an issue regarding CI, and what runs on what.

**Comment by dkindt** [comment]
@corevo I know this is quite the delayed response. I would like to "claim" this issue/ticket. I'm currently working on it.
